operations/mimir/images.libsonnet

```diff
@@ -4,6 +4,9 @@
     memcached: 'memcached:1.6.17-alpine',
     memcachedExporter: 'prom/memcached-exporter:v0.6.0',
 
+    redis: 'redis:6.2-bullseye',
```

Comment:
We're testing against 7.0.7 in #4371

Reply:
Noted. I will update. I was just using the version from grafana/redis jsonnet.